tests/inputs/update_all_equilibria_backwards_compatibility.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+"""
```

Comment:
see previous note about moving this stuff to `devtools`

we could probably also combine the scripts, or just have a single script that takes a directory as an argument and updates all the files in that directory, then we just call it twice

Reply:
I combined the 2 files and moved them to devtools/